### DISCUSSION TITLE
Make the Add button more versatile

### DISCUSSION THREAD

**Discussion by ModProg** [GitJournal/GitJournal#866] [Feature Requests]
## Description

I would like the add-note button (the green/brown one) to open a context menu, where you can select the note-types and folders.

**Comment by vHanda** [comment]
There is a way to select note types from the bottom left. But yeah, folders might be nicer.

I have doubts if people are using the existing note type buttons. I'll add some statistics so we can get some real data.

**Comment by ModProg** [comment]
Here is the way Fair-Mail handles the respond button, and I really like it.
![grafik](https://user-images.githubusercontent.com/11978847/110244344-9649bb80-7f5e-11eb-8bd1-4279b3490d99.png)
